psi4/src/psi4/libsapt_solver/sapt0.cc

```diff
@@ -1615,12 +1605,8 @@ void SAPT0::oo_df_integrals() {
     free_block(B_p_BB);
 
     // Get fitting metric
-    auto metric = std::make_shared<FittingMetric>(elstbasis_);
-    metric->form_eig_inverse(options_.get_double("DF_FITTING_CONDITION"));
-    double **J_temp = metric->get_metric()->pointer();
-    double **J_mhalf = block_matrix(ndf_, ndf_);
-    C_DCOPY(ndf_ * ndf_, J_temp[0], 1, J_mhalf[0], 1);
-    metric.reset();
+    auto metric = get_metric(elstbasis_);
+    auto J_mhalf = metric->pointer();
```

Comment:
Is the `J_mhalf` not getting modified in any of these 3 uses? Removed code makes an indep copy and new code doesn't (that I see).

Reply:
Correct: remove code made an independent copy, and new code does not.

`J_mhalf` was not getting modified, so I saw no reason to bother with the copy.